scraper.py

```diff
@@ -119,6 +119,7 @@
 LYRIC_JSON_PATH = 'lyrics.json'
 SONG_LIST_PATH = 'song_titles.txt'
 
+access_token = # put your genius.com API access token here as a string
```

Comment:
`access_token` should be defined in `local.py` - I provide a `local.py.default` file for you to copy and make a `local.py` file from so that you can fill in the access token there! This line is unnecessary here.

Reply:
Ahhhh, never encountered this before so didn't know. (I'm mostly an iOS Engineer.) Removed!